### ISSUE TITLE
addon action blocked

### ISSUE THREAD

**Issue by sindaliza** [WeakAuras/WeakAuras-Companion#1719]
-->

**Operating System:**
- [ x] Windows
- [ ] Mac
- [ ] Linux

**Companion Version:**   4.1.3

**Companion App Console Log:**
```
console only comes up with a box that says "weakauras profiling" but is empty otherwise
totally possible that i am doing this part wrong.  
```

**WeakAuras Version:**
```
5.3.4
```

**WeakAuras Errors:**
```
1x [ADDON_ACTION_BLOCKED] AddOn 'WorldQuestTracker' tried to call the protected function 'ActionButton2:SetAttribute()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/ActionButton.lua"]:350: in function `UpdatePressAndHoldAction'
[string "@FrameXML/ActionButton.lua"]:412: in function `Update'
[string "@FrameXML/ActionButton.lua"]:362: in function `UpdateAction'
[string "@FrameXML/ActionButton.lua"]:821: in function `OnEvent'
[string "@FrameXML/ActionButton.lua"]:216: in function <FrameXML/ActionButton.lua:213>


1x [ADDON_ACTION_BLOCKED] AddOn 'WeakAuras' tried to call the protected function 'ActionButton2:SetAttribute()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/ActionButton.lua"]:350: in function `UpdatePressAndHoldAction'
[string "@FrameXML/ActionButton.lua"]:412: in function `Update'
[string "@FrameXML/ActionButton.lua"]:362: in function `UpdateAction'
[string "@FrameXML/ActionButton.lua"]:821: in function `OnEvent'
[string "@FrameXML/ActionButton.lua"]:216: in function <FrameXML/ActionButton.lua:213>

1x [ADDON_ACTION_BLOCKED] AddOn 'WeakAuras' tried to call the protected function 'MultiBarBottomRight:ShowBase()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `ShowBase'
[string "@FrameXML/ActionBar.lua"]:347: in function `UpdateVisibility'
[string "@FrameXML/ActionBar.lua"]:310: in function `ShowOverride'
[string "@FrameXML/ActionBar.lua"]:300: in function `SetShown'
[string "@FrameXML/MultiActionBars.lua"]:32: in function <FrameXML/MultiActionBars.lua:30>
[string "@FrameXML/MultiActionBars.lua"]:42: in function `MultiActionBar_Update'
[string "@FrameXML/ActionBarController.lua"]:215: in function `ValidateActionBarTransition'
[string "@FrameXML/ActionBarController.lua"]:161: in function `ActionBarController_UpdateAll'
[string "@FrameXML/ActionBarController.lua"]:66: in function <FrameXML/ActionBarController.lua:55>

1x [ADDON_ACTION_BLOCKED] AddOn 'WeakAuras' tried to call the protected function 'MultiBarBottomLeft:SetFrameStrata()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `SetFrameStrata'
[string "@FrameXML/ActionBar.lua"]:195: in function `UpdateFrameStrata'
[string "@FrameXML/ActionBar.lua"]:187: in function `SetShowGrid'
[string "@FrameXML/MultiActionBars.lua"]:61: in function `MultiActionBar_HideAllGrids'
[string "@FrameXML/SpellBookFrame.lua"]:493: in function <FrameXML/SpellBookFrame.lua:479>
[string "=[C]"]: in function `Hide'
[string "@FrameXML/UIParent.lua"]:2874: in function `SetUIPanel'
[string "@FrameXML/UIParent.lua"]:2954: in function `HideUIPanel'
[string "@FrameXML/UIParent.lua"]:2602: in function <FrameXML/UIParent.lua:2594>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/UIParent.lua"]:3216: in function `HideUIPanel'
[string "@SharedXML/SharedUIPanelTemplates.lua"]:153: in function <SharedXML/SharedUIPanelTemplates.lua:144>

Locals:
Skipped (In Encounter)

Locals:
Skipped (In Encounter)

```

**Describe the bug"
keep getting an error that requires a reload to release the game back to a playable state after turning off the addon especially on pull in raid/M+ bosses

**Expected behavior**

addon should not need to be turned off especially at the beginning of a boss fight 
**Screenshots**

**Include a .zip file containing:**
`%programfiles%\World of Warcraft\_retail_\WTF\Account\<AccountName>\SavedVariables\WeakAuras.lua`
AND
`%appdata%\weakauras-companion\config.json`
[WeakAuras.zip](https://github.com/WeakAuras/WeakAuras-Companion/files/10420244/WeakAuras.zip)

**Comment by InfusOnWoW** [comment]
Nothing in WeakAuras is calling those functions, unless you have some very broken auras installed.

Also the game is known to be pretty buggy. Some of those bugs are fixed in the 10.0.5 release

**Comment by sindaliza** [comment]
awesome thank you for the help!!  i did have some old shadowlands auras
installed that i deleted but i use very few to begin with.  i will keep an
eye out for updates!
